apps/meteor/server/lib/findUsersOfRoom.ts

```diff
@@ -40,12 +40,12 @@ export function findUsersOfRoom({
 		...(limit > 0 && { limit }),
 	};
 
-	const searchFields = settings.get<string>('Accounts_SearchFields').trim().split(',');
+	const searchFields = settings.get<string>('Accounts_SearchFields').trim().split(',') as Array<keyof IUser>;
 
-	return Users.findPaginatedByActiveUsersExcept(filter, undefined, options, searchFields, [
+	return Users.findPaginatedByActiveUsersExcept(filter, [], options, searchFields, [
```

Comment:
Do undefined and [] work the same? 👀

Reply:
*Yeah*

This one, or rather **these** ones I am not so sure about; it's already a bad union (old code baby), type and type[] ? then adding another falsy undefined. the previous implementation of that method did a null check and default set that to a list any way. So, I moved that default here. 

Open to other POVs though.

Again, we don't have standards for these (at least not that I know of), like when to use null/undefined vs when to use type-specific falsy inputs like empty lists or empty strings etc. The rule that **I** follow or like to follow is any serializable data, use the type specific falsy, for objects "alive" objects or function types, use undef/null, with exceptions if any.

Reply:
There are other places that kinda does this mix as well iirc

Reply:
Not really sure 🤔 I mean, you're right that there's no standard, but what feels better? 👀 
@sampaiodiego , wanna put an standard here? :trollface:

Reply:
To me it's like english (or any language really), a parameter equals a question; for example in this case, `what are the username exceptions?` - the answer would be `none`, i.e. empty list, not `undefined`, i.e. "i don't know".

Reply:
My thought process, if it helps:

To me it's like english (or any language really), a parameter equals a question; for example in this case, "what are the username exceptions?" - the answer would be `no exceptions`, i.e. `empty list`, not `undefined`, i.e. `i'm not sure`.

it also enforces others to be more alert of **what** they're passing to a function.

Reply:
PS: i should mention the memory overhead, although I don't know how much, but from that perspective undefined is more "efficient" than empty lists or objects. But then again, would it matter *so much*? IDK 🙈